src/cli/configure/processing/ignoreFile.ts

```diff
@@ -1,27 +1,13 @@
-const splitIntoSanitisedLines = (str: string) =>
-  str
-    .trim()
-    .split(/\r?\n/)
-    .map((line) => line.trim());
-
 export const mergeWithIgnoreFile = (templateFile: string) => (
   inputFile?: string,
 ) => {
-  const templateLines = splitIntoSanitisedLines(templateFile);
-
-  const templateSet = new Set(templateLines.filter((line) => line !== ''));
-
-  const inputLines = splitIntoSanitisedLines(inputFile ?? '').filter(
-    (line) => !templateSet.has(line),
+  const replacedSkubaSection = inputFile?.replace(
+    /# managed by skuba[\s\S]*# end managed by skuba/,
+    '',
```

Comment:
Can we do the replacement inline here?

How about inverting this to be more like:

```typescript
if (typeof inputFile === 'undefined') {
  return templateFile;
}

return inputFile.replace('...', templateFile);
```

You could even one-liner it, but we might add some extra hacks in here later to make the migration less jarring.